ts/components/ui/FooterWithButtons.tsx

```diff
@@ -25,12 +26,12 @@ type OwnButtonProps = {
 
 type FooterButtonProps = ComponentProps<Button> & OwnButtonProps;
 
-interface SingleButton {
+export interface SingleButton {
   type: "SingleButton";
   leftButton: FooterButtonProps;
 }
 
-interface TwoButtonsInlineHalf {
+export interface TwoButtonsInlineHalf {
```

Comment:
There is no code inside FooterWithButtons that reacts when type is _**TwoButtonsInlineHalf**_
If so, we could remove it?

Reply:
Actually the code use it as "default option": if the `type` attribute is not equal to `"SingleButton"` nor `"TwoButtonsInlineThird"`, the component is rendered giving an half of the screen width to each button